src/bundle/ui-dev/src/modules/multi-file-upload/components/progress-bar/progress.bar.component.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const { Translator } = window;
+import { getTranslator } from '@ibexa-admin-ui/src/bundle/Resources/public/js/scripts/helpers/context.helper';
```

Comment:
Is there any possibility to make this import shorter 🤔

Reply:
we can create alias like @ibexa-admin-ui but probably in 5.0 :) because we have to clean up helpers